daly-bms-uart.h

```diff
@@ -143,9 +161,9 @@ class Daly_BMS_UART
     bool Init();
 
     /**
-     * send data to the bms and update the values
+     * @brief Update all the values in the get struct
      */
-    bool update();
```

Comment:
Sure its a good Idea? The Most of arduino librays usw This Name / function to Fall a global update

Reply:
I agree with you